zerver/lib/markdown/api_arguments_table_generator.py

```diff
@@ -125,6 +125,9 @@ def render_table(self, arguments: Sequence[Mapping[str, Any]]) -> List[str]:
             example = ""
             if "example" in argument:
                 example = argument["example"]
+                # Converting camel-case boolean to lower case.
+                if str(example) == "True" or str(example) == "False":
+                    example = str(example).lower()
```

Comment:
Thanks for figuring out where we need to make a change!  After thinking about this, I think this implementation is kinda hacky, and went with something different with more comments explaining the reasoning.  See the commit I pushed to this PR.  Thanks @ArunSankarKs!